tests/codes/test_param.py

```diff
@@ -55,3 +58,66 @@ def test_value_change(self):
 
     def test_tofrom_dict(self):
         assert self.pm == ParameterMapping.from_dict(self.pm.to_dict())
+
+
+@dataclass
+class MyDC:
+    a: int = 1
+    b: str = "hello"
+    c: bool = True
+
+
+mappings = {
+    "A": ParameterMapping("a", send=True, recv=False, unit="m"),
+    "B": ParameterMapping("b", send=True, recv=False, unit=""),
+    "C": ParameterMapping("c", send=True, recv=False, unit=""),
+    "D": ParameterMapping("d", send=False, recv=False, unit="cm"),
+    "E": ParameterMapping("e", send=False, recv=False, unit=""),
+    "F": ParameterMapping("f", send=False, recv=False, unit=""),
+}
+
+
+@dataclass
+class MyPF(MappedParameterFrame):
+    A: Parameter[float]
+    B: Parameter[str]
+    C: Parameter[bool]
+    D: Parameter[float]
+    E: Parameter[str]
+    F: Parameter[bool]
+
+    _mappings = mappings
+    _defaults = MyDC()
+
+    @property
+    def defaults(self) -> MyDC:
+        """Defaults for Plasmod"""
+        return self._defaults
+
+    @classmethod
+    def from_defaults(cls) -> MappedParameterFrame:
+        default_dict = asdict(cls._defaults)
+        default_dict["d"] = 0
+        default_dict["e"] = "0"
+        default_dict["f"] = False
+        return super().from_defaults(default_dict)
+
+
+class TestDefaultPM:
+    def test_mapped_default_pm_sets_unknown_values(self):
```

Comment:
Would `unmapped` be more descriptive than `unknown` here? Similar in the test below